book/build-svg.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -e
+
+cd "$(dirname "$0")"
+
+make -j"$(nproc)" svg 
+
+[[ -z "$CI" ]] && git diff --exit-code
```

Comment:
I'd expand this out to the following, just to be a little more readable
```bash
if [[ -n $CI ]]; then
  # In CI confirm that no svgs need to be built 
  git diff --exit-code
fi
```

Reply:
Fixed!